config/main.py

```diff
@@ -1852,27 +1853,70 @@ def override_config_table(db, input_config_db, dry_run):
 
     config_db = db.cfgdb
 
+    # Read config from configDB
+    current_config = config_db.get_config()
+    # Serialize to the same format as json input
+    sonic_cfggen.FormatConverter.to_serialized(current_config)
+
+    updated_config = update_config(current_config, config_input)
+
+    yang_enabled = is_yang_config_verification_enabled(config_db)
+    if yang_enabled:
+        # The ConfigMgmt will load YANG and validate
+        # runnning config during initialization.
+        try:
+            cm = ConfigMgmt()
```

Comment:
>ConfigMgmt [](http://example.com/codeflow?start=17&length=10)

There is an optional parameter `allowTablesWithoutYang=True`. Should it be False?

Reply:
I prefer no. That could be set to `False` when YANG covers all CONFIG_DB. Otherwise it will fail if CONFIG_DB has config without yang.